docs/ref/django-admin.txt

```diff
@@ -1065,6 +1065,30 @@ Mails the email addresses specified in :setting:`ADMINS` using
 
 Starts the Python interactive interpreter.
 
+All models from all installed apps are automatically imported. Furthermore, a
+set of default automatic imports to assist making queries, to change your
+project's settings and check datetime were added:
+
+* ``from datetime import date, datetime, timezone``
+* ``from django.db.models import *``
+* ``from django.db.models.functions import *``
+* ``from django.conf import settings``
```

Comment:
I think instead of having to list everything, we should emphasize that you can see all imports with  `--verbosity of 2 and above`, see that https://github.com/django/django/pull/18560/files#r1761486727
I'm thinking this list could change regularly and get very long and it would be easier if we don't need to add a "versionchanged" note with every new function added :thinking: 

https://github.com/django/django/pull/18560/files#r1761486727 still needs addressing for this to work

Reply:
So, do you mean to also list the imports of modules?